pyiron_atomistics/atomistics/structure/structurestorage.py

```diff
@@ -212,9 +212,7 @@ def add_structure(self, structure, identifier=None, **arrays):
         if structure.has("initial_magmoms"):
             arrays["spins"] = structure.spins
         if "selective_dynamics" in structure.get_tags():
```

Comment:
Should this also use the `.has` syntax now?

Reply:
That would definitely be a possibility, but since `selective_dynamics` doesn’t exist in ASE, and the same functionality seems to be implemented in a fairly different way, (`FixAtoms`?) it might also make sense to leave it like this for now.